source/slang/hlsl.meta.slang

```diff
@@ -105,13 +105,13 @@ struct ByteAddressBuffer
     [__readNone]
     [ForceInline]
     [require(cpp_cuda_glsl_hlsl_metal_spirv, byteaddressbuffer)]
-    uint Load(int location)
+    uint Load(int location, int aligned)
```

Comment:
You can't change the signature of public facing method here. Instead, just pass 0 to `__byteAddressBufferLoad` call in the body. Same applies to other methods.

Reply:
done